README.md

```diff
@@ -40,6 +40,7 @@
 
 ## GitHub Actions for Readmes
 
+- [activity-log](https://github.com/TheDanniCraft/activity-log#readme) - Dynamically display your recent github activity (Markdown  or HTML).
```

Comment:
Hey @gltched-usr, to keep the name style consistent please update the name to "Activity Log" instead of "activity-log".